domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/build.gradle

```diff
@@ -20,7 +22,9 @@ dependencies {
     //TODO remove with 1.0.0
     implementation prefixedProject('api-cc-en_GB-jvm')
 
+    testImplementation spekExtensions()
     testImplementation prefixedProject('verbs-internal-jvm')
+    testImplementation prefixedProject('translations-en_GB-jvm')
```

Comment:
I guess this is not required

Reply:
The translations are not available and compilation fails without this line.

Reply:
The actual problem is that this project depends on `verbs-internal-jvm` instead of `specs-jvm`. Could you please change that (was a failure on my part)? spekExtensions and translations are then not required anymore

Reply:
Done.

Reply:
Done.